app/src/main/java/com/aguesoftguar/medalarm/addeditmedicine/AddEditMedicineFragment.java

```diff
@@ -329,4 +332,17 @@ public void onClick(View view) {
    @Override public boolean isActive() {
       return isAdded();
    }
+
+   @Override
+   public void showPatients(List<Patient> patients) {
```

Comment:
Maybe, the name of this method is not correct because the patients don't show when this method is called.